docs/tasks.md

```diff
@@ -5,6 +5,7 @@ Tasks enable file-centric workflows in Box. User can create tasks on files and a
 
 * [Create a Task](#create-a-task)
 * [Get a Task's Information](#get-a-tasks-information)
+* [Delete a Task Permanently](#delete-a-task-permanently)
```

Comment:
Tasks cannot be temporarily deleted; why do we need to add this qualifier throughout?